diff --git a/include/configs/rockchip-common.h b/include/configs/rockchip-common.h
index f1f22ff..4e513f2 100644
--- a/include/configs/rockchip-common.h
+++ b/include/configs/rockchip-common.h
@@ -143,10 +143,8 @@
 	"boot_fit;"
 #else
 #define RKIMG_BOOTCOMMAND			\
-	"boot_android ${devtype} ${devnum};"	\
-	"boot_fit;"				\
-	"bootrkp;"				\
-	"run distro_bootcmd;"
+	"run distro_bootcmd;"	\
+	"bootrkp;"
 #endif
 
 #endif /* CONFIG_SPL_BUILD */
